src/client/profileKeys.js

```diff
@@ -0,0 +1,69 @@
+const Https = require('https')
+const crypto = require('crypto')
+
+function makePrivateKey (data) {
+  let pem = '-----BEGIN PRIVATE KEY-----\n'
+  const suf = '-----END PRIVATE KEY-----\n'
+  let raw = data.split('\n').slice(1, -2).join('')
+  while (raw.length > 0) {
+    pem += raw.substring(0, 65) + '\n'
+    raw = raw.substring(65)
+  }
+
+  return crypto.createPrivateKey(pem + suf)
+}
+
+function makePublicKey (data) {
+  let pem = '-----BEGIN PUBLIC KEY-----\n'
+  const suf = '-----END PUBLIC KEY-----\n'
+  let raw = data.split('\n').slice(1, -2).join('')
+  while (raw.length > 0) {
+    pem += raw.substring(0, 65) + '\n'
+    raw = raw.substring(65)
+  }
+
+  return crypto.createPublicKey(pem + suf)
+}
+
+module.exports = function (client, options) {
+  if (!options.accessToken) throw new Error('Invalid user access token')
+  return new Promise((resolve, reject) => {
+    const req = Https.request('https://api.minecraftservices.com/player/certificates', {
```

Comment:
This is already handled in prismarine-auth

Reply:
Alright. My bad. It didn't seem to work when I ran online tests. It will likely prompt a few changes but removing that file won't affect the signing aspect.